src/components/Snackbar/style.ts

```diff
@@ -10,7 +10,24 @@ const useStyles = makeStyles(() => ({
     borderColor: colors.green.snackbar,
     borderRadius: 10,
     padding: 10,
-    margin: 2
+    fontSize: 20,
+    lineHeight: '30px',
+    maxWidth: 500,
+    maxHeight: 65,
+    paddingBottom: 12,
+    '& SVG': {
+      marginTop: 4,
+      [theme.breakpoints.down('xs')]: {
+        marginTop: 2
+      }
+    },
+    [theme.breakpoints.down('xs')]: {
+      fontSize: 11,
```

Comment:
about all font sizes: use our predefined typography where it is possible - look at styles of other components